### DISCUSSION TITLE
Merging PRs

### DISCUSSION THREAD

**Discussion by lolodomo** [openhab/openhab-addons#14707] [Add-on Maintainers Discussions]
Hi all openHAB maintainers,
I started today reviewing and merging few PRs but I have now suddenly a doubt. Should we have a different maintainer for approving a PR and merging it ?

**Comment by hmerk** [comment]
No, this has been changed in April to speed up the review process. There is a longer post from Kai…
https://github.com/orgs/openhab/teams/add-ons-maintainers/discussions/11
